samples/build-calcupad-pipeline.yaml

```diff
@@ -2,24 +2,44 @@
 # This example relies on the git-clone Task:
 # https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.2
 #
+# This example shows how to use a PersistentVolumeClaim to store the artifacts on a NFS.
+# When using a PVC, an existing PersistentVolume is required.
+# Modify the following for the PV if you wish to use a NFS:
+#   server: <your-nfs-server-ip>
+#   path: <your-nfs-path>
+#
 # ###
 #
 # This Pipeline will first clone a git repository into a shared workspace.
 # The workspace will then be copied to the Orka VM, and the supplied build
 # script will be executed inside the VM.
 # Build artifacts will then be transferred back to the Tekton workspace,
 # which can be consumed by a later Task in the Pipeline, or simply
-# saved to a cloud storage bucket.
+# saved to a mounted NFS, cloud storage bucket, etc.
 #
 # ###
 # Install the git-clone task:
 # kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/git-clone/0.2/git-clone.yaml
 # ###
 ---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: nfs-pv
+spec:
+  capacity:
+    storage: 1Gi
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: manual
+  nfs:
+    server: <your-nfs-server-ip>
+    path: <your-nfs-path>
+---
```

Comment:
I think we can leave out the example PV.

In general, folks will likely already have a storage solution if they are using Tekton for their CI/CD. The NFS example is a bit specific to how we set up Tekton with Orka on our own infra (which is not something typical users will be doing).

Just demonstrating how to use the PV for the Tekton workspace to save your macOS build artifacts in should be enough